mne/stats/tests/test_cluster_level.py

```diff
@@ -469,7 +469,8 @@ def test_cluster_permutation_with_adjacency(numba_conditional, monkeypatch):
                 X1d_3, adjacency=adjacency, threshold=dict(start=10, step=1)
             )
         if not did_warn:
-            assert len(w) == 1
+            messages = [str(ww.message) for ww in w]
+            assert any("is more extreme" in message for message in messages), messages
```

Comment:
Snuck in a fix for #11931 since it was very small, mostly a maintenance thing (no need for `devel.rst` update), and we'll want both changes backported